Lib/test/test_capi/test_misc.py

```diff
@@ -777,33 +777,17 @@ def test_pytype_fromspec_with_repeated_slots(self):
                 with self.assertRaises(SystemError):
                     _testcapi.create_type_from_repeated_slots(variant)
 
-    @warnings_helper.ignore_warnings(category=DeprecationWarning)
     def test_immutable_type_with_mutable_base(self):
-        # Add deprecation warning here so it's removed in 3.14
-        warnings._deprecated(
-            'creating immutable classes with mutable bases', remove=(3, 14))
-
         class MutableBase:
             def meth(self):
                 return 'original'
 
-        with self.assertWarns(DeprecationWarning):
-            ImmutableSubclass = _testcapi.make_immutable_type_with_base(
-                MutableBase)
-        instance = ImmutableSubclass()
-
-        self.assertEqual(instance.meth(), 'original')
-
-        # Cannot override the static type's method
-        with self.assertRaisesRegex(
-                TypeError,
-                "cannot set 'meth' attribute of immutable type"):
-            ImmutableSubclass.meth = lambda self: 'overridden'
-        self.assertEqual(instance.meth(), 'original')
+        with self.assertRaisesRegex(TypeError, 'Creating immutable type'):
+            _testcapi.make_immutable_type_with_base(MutableBase)
 
         # Can change the method on the mutable base
         MutableBase.meth = lambda self: 'changed'
-        self.assertEqual(instance.meth(), 'changed')
+        self.assertEqual(MutableBase().meth(), 'changed')
```

Comment:
I don't think that this test is still relevant. You can remove it. Or if you want to keep it, maybe mutable the classe before calling make_immutable_type_with_base()?

Reply:
Thanks, addressed! 👍